packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -2,11 +2,10 @@
     import { fade } from 'svelte/transition'
     import { Button, DeveloperIndicatorPill, HR, Icon, Modal, Text, Toggle } from 'shared/components'
     import { localize } from '@core/i18n'
-    import { LocaleArguments } from '@core/i18n/types'
-    import { ledgerDeviceState } from 'shared/lib/ledger'
+    import { LedgerConnectionState } from '@lib/typings/ledger'
+    import { ledgerDeviceStatus, getLedgerDeviceStatus } from 'shared/lib/ledger'
```

Comment:
We've adjusted the codebase quite a bit from V1 - now we're trying to slowly whittle down all code that's not in `lib/core`, `lib/contexts` nor `lib/auxiliary`. Perhaps we can refactor the code in `shared/lib/ledger.ts` to a new `core/ledger` module?